.gitignore

```diff
@@ -23,3 +23,5 @@ swift/Sources/Generated
 # Code coverage files
 coverage.info
 coverage/
+
+cmake-build-debug/
```

Comment:
What is this for?

Reply:
It's the default location of the generated CMake files in `CLion`. That is to say, when a cmake project run in `CLion`, this folder will be created automatically to contain Makefile.

Reply:
ok 👌